ui/components/connections/ConnectionTable.js

```diff
@@ -457,9 +457,14 @@ const ConnectionTable = ({
 
   const handleStatusChange = async (e, connectionId, connectionKind) => {
     e.stopPropagation();
+    const status = e.target.value.toUpperCase();
+    let subtitle = `Are you sure that you want to transition the connection status to ${status}?`;
+    if (status === 'DISCONNECTED') {
```

Comment:
This is inappropriate and doesn't consider other connection types. The connection to be transitioned should be checked for type.